bundle/edu.gemini.itc.web/src/main/java/edu/gemini/itc/web/html/AcqCamPrinter.java

```diff
@@ -0,0 +1,74 @@
+package edu.gemini.itc.web.html;
+
+import edu.gemini.itc.acqcam.AcqCamRecipe;
+import edu.gemini.itc.acqcam.AcquisitionCamParameters;
+import edu.gemini.itc.acqcam.AcquisitionCamera;
+import edu.gemini.itc.shared.ImagingResult;
+import edu.gemini.itc.shared.Parameters;
+
+import java.io.PrintWriter;
+
+/**
+ * Helper class for printing Acquisition Camera calculation results to an output stream.
+ */
+public final class AcqCamPrinter extends PrinterBase {
```

Comment:
Yeah, I was thinking about that. However these "Printer" classes are just containers for extracted code that was somewhere mixed into the calculation code. In the end it was easier to just copy the existing Java code into Java classes instead of going through the hoops to change it into Scala. So the short answer is: I am just lazy.